dotnet/src/SemanticKernel.Abstractions/AI/TextToImage/ITextToImageService.cs

```diff
@@ -14,19 +15,17 @@ namespace Microsoft.SemanticKernel.TextToImage;
 public interface ITextToImageService : IAIService
 {
     /// <summary>
-    /// Generate an image matching the given description
+    /// Generate images matching the given text description
```

Comment:
`/// Given a prompt and/or an input image, the model will generate a new image.`